src/main/java/edu/umich/kanban/XmlMerge.java

```diff
@@ -44,7 +44,6 @@ public String mergeFiles(String queryString, String xsltPath, String xmlPath) th
       
       String its=queryJiraITS(queryString);
       String sakai = queryJiraSakai(queryString);
-      String internet2 = queryJiraInternet2(queryString);
```

Comment:
It seems like the two remaining methods, `queryJiraITS()` and `queryJiraSakai()` are very similar and should be combined into a single method in a new class, `JiraQuery`, for example.